src/main/java/duke/exceptions/MissingDeadlineParamException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exceptions;
+
+/**
+ * Custom Exception to be used when there is a Missing Parameter for the Deadline Command is encountered
+ */
+public class MissingDeadlineParamException extends DukeException {
+    /**
+     * Returns a String representation of the exception
+     * @return A String representation of the exception
+     */
+    @Override
+    public String toString() {
```

Comment:
An alternative way of specifying the error message is:
Passing the message to the constructor e.g. `throw new MissingDeadlineParamException("OOPS! ...")`
You can access the error message by doing "exception.getMessage()" in the catch block